src/app/config/urls.ts

```diff
@@ -43,7 +43,7 @@ export const LocationToPath = {
   [Location.UniParcEntry]: '/uniparc/:accession',
   [Location.UniParcResults]: '/uniparc',
   [Location.ProteomesEntry]: '/proteomes/:accession',
-  [Location.ProteomesResults]: '/proteomes',
+  [Location.ProteomesResults]: `/${Namespace.proteomes}`,
```

Comment:
I am think tying everything to the `Namespace` enum would be a good thing to do more generally in our urls.

Reply:
Good idea. In that case, do it for everything in this file of URLs.